packages/lesswrong/components/forumEvents/ForumEventFrontpageBanner.tsx

```diff
@@ -185,6 +185,7 @@ const styles = (theme: ThemeType) => ({
     top: -HEADER_HEIGHT,
     right: 0,
     width: '100%',
+    height: `calc(100% + ${HEADER_HEIGHT}px)`
```

Comment:
This is because the image is too wide so would not cover all of the header vertically on mobile. @agnesstenlund if you could get an image that is ~1509 × 1679 we could revert this (which might be better because it would stretch the image in future events)